prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -833,6 +849,44 @@ class Hl7Serializer(val metadata: Metadata): Logging {
         }
     }
 
+    /**
+     * Decode an XTN (e.g. phone number) component of an HL7 message.
+     * @param terser the HL7 terser
+     * @param element the element to decode
+     * @return the phone number or empty string
+     */
+    internal fun decodeXTNPhoneNumber(terser: Terser, element: Element): String {
+        var decodedPhoneNumber = ""
```

Comment:
How does this handle repetition in the phone number field?

Reply:
Updated code to handle repetition.  See new changes to code and unit test.